test/CLITest.ts

```diff
@@ -187,4 +187,49 @@ describe('CLI', function() {
       '3 + 4;'
     );
   });
+
+  it('can load plugins written with ES modules by default', async function() {
+    let afile = await createTemporaryFile('a-file.js', '3 + 4;');
+    let { status, stdout, stderr } = await runCodemodCLI([afile, '-p', plugin('increment-export-default')]);
+
+    deepEqual(
+      { status, stdout, stderr },
+      {
+        status: 0,
+        stdout: `${afile}\n1 file(s), 1 modified, 0 errors\n`,
+        stderr: ''
+      }
+    );
+    strictEqual(
+      await readFile(afile, 'utf8'),
+      '4 + 5;'
+    );
+  });
+
+  it('can load plugins with multiple files with ES modules by default`', async function() {
+    let afile = await createTemporaryFile('a-file.js', '3 + 4;');
+    let pluginFile = join(__dirname, `fixtures/plugin/increment-export-default-multiple/increment-export-default.js`);
+    let { status, stdout, stderr } = await runCodemodCLI([afile, '-p', pluginFile]);
+
+    deepEqual(
+      { status, stdout, stderr },
+      {
+        status: 0,
+        stdout: `${afile}\n1 file(s), 1 modified, 0 errors\n`,
+        stderr: ''
+      }
+    );
+    strictEqual(
+      await readFile(afile, 'utf8'),
+      '4 + 5;'
+    );
+  });
+
+  it('fails when specifying --use-local-babel as there are no plugins loaded', async function() {
+    let afile = await createTemporaryFile('a-file.js', '3 + 4;');
+    let { status } = await runCodemodCLI([afile, '-p', plugin('increment-export-default'), '--use-local-babel']);
```

Comment:
We should also make some sort of assertion about the output. Ideally we could even catch `SyntaxError` and provide a better error message telling users they may need to use a custom `.babelrc`.

Reply:
It's difficult to find where to place the try catch to determine if the SyntaxError is a result of the plugin or the code that we're modding.  We could also look at the stack trace to determine where the syntax error occurred?:

```
/Users/z002s72/dev/git-clones/babel-codemod/test/fixtures/plugin/increment-export-default.js:1
(function (exports, require, module, __filename, __dirname) { export default function() {
                   ^^^^^^

SyntaxError: Unexpected token export
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:599:28)
    at loader (/Users/z002s72/dev/git-clones/babel-codemod/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/z002s72/dev/git-clones/babel-codemod/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
```

Reply:
Looks like wrapping the following in try/catch captures the specific plugin related SyntaxError:

``` 
// index.ts, line 76
let plugins = options.getBabelPlugins();
```

However the error is undefined...im little lose there :)

Reply:
I’ll take a look when I get home.